qiskit_dynamics/backend/dynamics_backend.py

```diff
@@ -109,9 +110,7 @@ class DynamicsBackend(BackendV2):
       indicating solver methods and options. Defaults to the empty dictionary ``{}``.
     * ``subsystem_dims``: Dimensions of subsystems making up the system in ``solver``. Defaults to
       ``[solver.model.dim]``.
-    * ``subsystem_labels``: Integer labels for subsystems. Defaults to ``[0, ...,
-      len(subsystem_dims) - 1]``.
-    * ``meas_map``: Measurement map. Defaults to ``[[idx] for idx in subsystem_labels]``.
+    * ``meas_map``: Measurement map. Defaults to ``[[idx] for idx in range(len(subsystem_dims))]``.
```

Comment:
[Q] What is `meas_map` in dynamicsbackend used for ?

Reply:
I always forget exactly how `meas_map` works. `DynamicsBackend` itself doesn't use `meas_map`, but I think it's required by transpilation and/or pulse schedule validation. I believe the default `meas_map` described here allows measurements of any subset of qubits to pass the validation checks.

Reply:
The `meas_map` is used for mapping qubits which have to be measured concurrently. The configuration of `meas_map` used here indicates each qubit can be measured independently.

Just asking, I just understood the need for transpile and pulse validation. Thanks.